tests/test_Functional_API.py

```diff
@@ -18,4 +18,9 @@ def test_get_api_version(self):
         zapi = ZabbixAPI(url='http://127.0.0.1',
                          user='Admin',
                          password='zabbix')
-        self.assertEqual(zapi.api_version(), '3.0.1')
+        if os.environ['ZABBIX_VERSION'] == '3':
+            self.assertEqual(zapi.api_version(), '3.0.1')
+        elif os.environ['ZABBIX_VERSION'] == '2':
+            self.assertEqual(zapi.api_version(), '2.4.7')
```

Comment:
The short answer is Yes.
The only thing that I didn't write these containers myself. And just use
those that I found and tested.
In the repository there is no version earlier than 2.4. Accordingly, it is
necessary to write my container implementation, or to look in other
repositories.


С уважением, Сёмочкин Максим.
*Моб*: +7(926) 905-6085
*Twitter*: maksim77 <https://twitter.com/maksim77>
*Skype*: maksim77ster

2016-03-25 7:55 GMT+03:00 Alexey Dubkov <notifications@github.com>:

> In tests/test_Functional_API.py
> <https://github.com/blacked/py-zabbix/pull/39#discussion_r57419241>:
>
> > @@ -18,4 +18,9 @@ def test_get_api_version(self):
> >          zapi = ZabbixAPI(url='http://127.0.0.1',
> >                           user='Admin',
> >                           password='zabbix')
> > -        self.assertEqual(zapi.api_version(), '3.0.1')
> > +        if os.environ['ZABBIX_VERSION'] == '3':
> > +            self.assertEqual(zapi.api_version(), '3.0.1')
> > +        elif os.environ['ZABBIX_VERSION'] == '2':
> > +            self.assertEqual(zapi.api_version(), '2.4.7')
>
> Could we run test also on version 2.2.4 or more older? There was a changes
> in api between this two versions.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/blacked/py-zabbix/pull/39/files/02a5bd4a5bcf4fdfdd498618f140394bbdbdfc9f#r57419241>
>